src/js/components/organisms/colors/colors.vue.html

```diff
@@ -0,0 +1,28 @@
+<article class="colors">
```

Comment:
i'm a bit confused about two things:

1. is it an organism or a molecule?
1. should it be its own organism or a molecule/organism inserted into an overview organism (src/js/components/organisms-page/overview/overview.html)?

seems to me we should be re-using the existing overview component, which is the basis for all of our organism components